src/main/scala/beam/sim/BeamHelper.scala

```diff
@@ -140,7 +141,19 @@ trait BeamHelper extends LazyLogging {
             )
           ) ++ {
             if (parsedArgs.useCluster)
-              Map("beam.cluster.clusterType" -> parsedArgs.clusterType.get.toString)
+              Map(
+                "beam.cluster.clusterType" -> parsedArgs.clusterType.get.toString,
+                "akka.actor.provider"      -> "akka.cluster.ClusterActorRefProvider",
+                "akka.remote.artery.canonical.hostname" -> parsedArgs.nodeHost.getOrElse(
+                  throw new RuntimeException("`nodeHost` is not set")
```

Comment:
It probably doesn't hurt to have this as a backup. Note that it isn't necessary though - the args parser handles this already - Line 101. Same for nodePort and seedAddress